defaults/main/template.yml

```diff
@@ -360,25 +360,35 @@ nginx_config_http_template:
 nginx_config_status_enable: false
 nginx_config_status_template_file: http/status.conf.j2
 nginx_config_status_file_location: /etc/nginx/conf.d/status.conf
-nginx_config_status_log: false
-nginx_config_status_port: 8080
-nginx_config_status_allow: 127.0.0.1
-nginx_config_status_deny: all
+nginx_config_status_port: 8080  # Optional -- Defaults to 8080
+nginx_config_status_access_log:  # Optional -- Set to 'false' to disable access log
+  location: /var/log/nginx/access.log  # Required
+  name: main  # Required
+nginx_config_status_allow:  # Optional
```

Comment:
Hi @alessfg,

There'is a few missing `BREAKING CHANGES` for the all the variable converted to list.

For examples `nginx_config_status_allow`, `nginx_config_status_deny`, `*_allow`, `*_deny`.

If we don't change our vars we'ill get : 

```diff
--- before: /etc/nginx/conf.d/status.conf
+++ after: /root/.ansible/tmp/ansible-local-1037fmtytlo/tmpcb2bqvtq/status.conf.j2
@@ -4,10 +4,20 @@
 
 server {
     listen 8080;
-    access_log off;
+    access_log /var/log/nginx/access.log main;
     location /nginx_status {
         stub_status on;
-        allow 127.0.0.1;
-        deny all;
     }
+    allow 1;
+    allow 2;
+    allow 7;
+    allow .;
+    allow 0;
+    allow .;
+    allow 0;
+    allow .;
+    allow 1;
+    deny a;
+    deny l;
+    deny l;
 }
```

Then breaking the handler `nginxinc.nginx_core.nginx_config : (Handler - NGINX Config) Print NGINX error if syntax check fails`.

Thanks,

Reply:
Aaaaand done 😄